mmv1/products/cgc/terraform.yaml

```diff
@@ -51,6 +51,13 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           deletion_protection: "false"
         ignore_read_extra:
           - "deletion_protection"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "sql_sqlserver_instance_authorized_network"
+        primary_resource_id: "default"
```

Comment:
We will probably need to add `primary_resource_type: "google_sql_database_instance"` for snippets for handwritten resources